sylvia-derive/src/parser.rs

```diff
@@ -262,6 +263,35 @@ fn parse_generics(content: &ParseBuffer) -> Result<Vec<Path>> {
     Ok(params)
 }
 
+fn interface_has_custom(content: ParseStream) -> Result<(bool, bool)> {
```

Comment:
Going with the `Customs` I previously mentioned would make this signature far more readable - maybe it is actually a good idea already?